Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackWidget.swift

```diff
@@ -151,6 +58,57 @@ struct SentryUserFeedbackWidget {
             return result
```

Comment:
`m`: I have implemented a `PassThroughWindow` in the past (I'll share it with you internally in Slack) and encountered an edge case where events did not correctly pass through between iOS 17 and iOS 18. 

Here's a Apple Developer Forum thread I would like you to read through and double-check it does/does not apply to this hit test logic.

https://developer.apple.com/forums/thread/762292

Reply:
I did take a look through that thread, I hadn't seen it before, but this is out of scope for this PR. The code here is not new, it's already shipped, it only showed up in the diff because due to the refactor it was outdented one scope level.

Worringly, we only run the ui tests on iOS 16.4 and 17.5 currently, but I actually just opened a separate PR to update the 17.5 run to 18.3:

Reply:
See https://github.com/getsentry/sentry-cocoa/pull/5125